packages/notifications-library/src/notifications_library/_db.py

```diff
@@ -0,0 +1,63 @@
+import sqlalchemy as sa
+from models_library.products import ProductName
+from models_library.users import UserID
+from simcore_postgres_database.models.jinja2_templates import jinja2_templates
+from simcore_postgres_database.models.products_to_templates import products_to_templates
+from simcore_postgres_database.models.users import users
+from sqlalchemy.ext.asyncio import AsyncEngine
+
+
+class _BaseRepo:
+    def __init__(self, db_engine: AsyncEngine):
+        assert db_engine is not None  # nosec
+        self.db_engine = db_engine
+
+    async def _get(self, query):
+        async with self.db_engine.begin() as conn:
+            result = await conn.execute(query)
+            return result.first()
+
+
+class UsersRepo(_BaseRepo):
+    async def get_user_data(self, user_id: UserID):
+        """
+        Raises:
+            UserNotFoundError
+        """
+        return await self._get(
+            sa.select(
+                users.c.first_name,
+                users.c.last_name,
+                users.c.email,
+            ).where(users.c.id == user_id)
+        )
+
```

Comment:
question, why you do not use the UsersRepo in simcore-postgres-database library?